modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -13,13 +13,12 @@ You can integrate {product-title} with {ocp} built-in container image registry.
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
 . Under the *Image Integrations* section, select *Generic Docker Registry*.
-+
-The *Configure image integration* modal box opens.
-. Click *New Integration*.
+. Click *New integration*.
 . Enter the details for the following fields:
-.. *Integration Name*: The name of the integration.
-.. *Types*: Select *Registry*.
+.. *Integration name*: The name of the integration.
 .. *Endpoint*: The address of the registry.
 .. *Username* and *Password*.
-. Select *Test* (*`checkmark`* icon) to test that the integration with the selected registry is working.
-. Select *Create* (*`save`* icon) to create the configuration.
+. Select *Disable TLS certificate validation (insecure)* if you are not using a TLS certificate when connecting to the registry.
```

Comment:
If this is optional, it might make sense to flip it around ("If you are not using a TLS certificate when connecting to the registry, select *Disable TLS certificate validation (insecure)*.")